script/publish_static

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+import os
+import json
+from argparse import ArgumentParser
+
+from google.cloud import bigquery
+
+DATA_FILENAME = "data.csv"
+SCHEMA_FILENAME = "schema.json"
+DESCRIPTION_FILENAME = "description.txt"
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--data-dir",
+        default="templates/",
+        help="Path containing CSV's containing static data",
+    )
+    parser.add_argument("--project-id", help="Project to publish tables to")
+    return parser.parse_args()
+
+
+def load_table(
+    data_file_path, schema_file_path=None, description_file_path=None, project=None
+):
+    client = bigquery.Client()
+
+    # Assume path is .../dataset/table/data.csv
+    path_split = os.path.normcase(data_file_path).split("/")
+    dataset_id = path_split[-3]
+    table_id = path_split[-2]
```

Comment:
I acknowledge that the script is tightly coupled with the directory structure and this will cause problems if the structure changes in the future or if the `data.csv` is in the wrong place.  I haven't thought of a nice solution to this yet.

Reply:
This seems reasonable for now.